<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(169, 169, 169);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="appFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="headingFrame">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1281</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(235, 235, 235);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="labelOPCUA">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>10</y>
          <width>611</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(43, 100, 173);
font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="text">
         <string>OPC Unified Architecture Klient</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="logoSMC">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>191</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="pixmap">
         <pixmap>icones_for_gui/smc_logo.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="connectivityFrame">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>90</y>
         <width>231</width>
         <height>80</height>
        </rect>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="connectButton">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>0</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(235, 235, 235);
font: 14pt &quot;.AppleSystemUIFont&quot;;
color: rgb(43, 100, 173);
</string>
        </property>
        <property name="text">
         <string>Připojit</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="disconnectButton">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>40</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(235, 235, 235);
font: 14pt &quot;.AppleSystemUIFont&quot;;
color: rgb(43, 100, 173);
</string>
        </property>
        <property name="text">
         <string>Odpojit</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="aksEndpoints">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(235, 235, 235);
font: 14pt &quot;.AppleSystemUIFont&quot;;
color: rgb(43, 100, 173);
</string>
        </property>
        <property name="text">
         <string>Dotaz serveru</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="connectivityIcon">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="pixmap">
         <pixmap>icones_for_gui/disconnected.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QPlainTextEdit" name="statusBar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>640</y>
         <width>511</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(235, 235, 235);
font: 14pt &quot;.AppleSystemUIFont&quot;;
color: rgb(43, 100, 173);</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QFrame" name="URIFrame">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>441</width>
         <height>80</height>
        </rect>
       </property>
       <widget class="QLineEdit" name="serverURI">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>0</y>
          <width>301</width>
          <height>31</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(235, 235, 235);
font: 14pt &quot;.AppleSystemUIFont&quot;;
color: rgb(43, 100, 173);</string>
        </property>
        <property name="text">
         <string>opc.tcp://opuaserver.com:48010</string>
        </property>
       </widget>
       <widget class="QLabel" name="connectLabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;.AppleSystemUIFont&quot;;
color: rgb(43, 100, 173);</string>
        </property>
        <property name="text">
         <string>Připojení k serveru:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="mainFrame">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>1271</width>
         <height>461</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QTableView" name="dataChangeUI">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>0</y>
          <width>741</width>
          <height>461</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(235, 235, 235);
font: 14pt &quot;.AppleSystemUIFont&quot;;
color: rgb(43, 100, 173);</string>
        </property>
       </widget>
       <widget class="QTreeView" name="browserUI">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>511</width>
          <height>461</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(235, 235, 235);
font: 14pt &quot;.AppleSystemUIFont&quot;;
color: rgb(43, 100, 173);</string>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="QueryFrame">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>80</y>
         <width>401</width>
         <height>91</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>131</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_1">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>0</y>
          <width>271</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLineEdit" name="lineEdit_1">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_3"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
